src/ConnectedField.js

```diff
@@ -72,7 +71,10 @@ const createConnectedField = ({
         _value: ownProps.value // save value passed in (for checkboxes)
       }
     },
-    actions,
+    (dispatch, initialProps) => mapValues({ blur, change, focus }, actionCreator => (...args) => {
```

Comment:
Well that's a tradeoff, I could imagine some use-cases where user would be forced to change the dispatch over life time of the component. However, what we could rather do, was some memoization.